### PR TITLE
Use travel activity instead of t2f travel inside action point

### PR DESCRIPTION
**Pull request by roman-karpovich**
https://trello.com/c/Y2mnfRd2/15-please-make-action-point-model-by-travel-activity

### CODE REVIEW

File: src/etools/applications/action_points/tests/test_models.py [hunk 1]
```diff
@@ -55,7 +55,7 @@ def test_tpm_related(self):
         self.assertEqual(action_point.related_module, ActionPoint.MODULE_CHOICES.tpm)
 
     def test_t2f_related(self):
-        action_point = ActionPointFactory(travel=TravelFactory())
+        action_point = ActionPointFactory(travel_activity=TravelFactory().activities.first())
```
Comment (domdinicola) [inline src/etools/applications/action_points/tests/test_models.py]: why don't use TravelActivityFactory

File: src/etools/applications/action_points/tests/test_views.py [hunk 2]
```diff
@@ -108,8 +108,8 @@ def test_create_author(self):
         self.assertEqual(response.data['assigned_by']['id'], self.unicef_user.id)
 
     def test_create_t2f_related(self):
-        travel = TravelFactory()
-        data = {'travel': travel.id}
+        travel_activity = TravelFactory().activities.first()
```
Comment (domdinicola) [inline src/etools/applications/action_points/tests/test_views.py]: why don't use TravelActivityFactory?

File: src/etools/applications/action_points/tests/test_views.py [hunk 3]
```diff
@@ -119,18 +119,18 @@ def test_create_t2f_related(self):
...
         )
         self.assertEqual(response.status_code, status.HTTP_201_CREATED)
-        self.assertIsNotNone(response.data['travel'])
-        self.assertEqual(response.data['travel'], travel.id)
+        self.assertIsNotNone(response.data['travel_activity'])
+        self.assertEqual(response.data['travel_activity'], travel_activity.id)
 
     def test_travel_filter(self):
-        travel = TravelFactory()
+        travel_activity = TravelFactory().activities.first()
```
Comment (domdinicola) [inline src/etools/applications/action_points/tests/test_views.py]: why don't use TravelActivityFactory?

Comment (roman-karpovich) [reply src/etools/applications/action_points/tests/test_views.py]: updated

**Pull request merged**